### ISSUE TITLE
[Tracking] ONNX 1.8 Release

### ISSUE THREAD

**Issue by askhade** [onnx/onnx#2942]
Creating this issue to collect and track all the work items for ONNX 1.8 release.
The release is planned for October 2020.

@onnx/sig-archinfra , @onnx/sig-operators , @onnx/steering-committee , @sveta-levitan, @wschin , @chinhuang007 and community members, please add work items here so that we can scope the release.

Thanks!

**Comment by chinhuang007** [comment]
I'd like to share a couple of build/release items here so the community can help prioritize and eventually fix the issues.

1. Consolidate and automate release wheels building and upload to Pypi for Linux, OSX, and Windows, https://github.com/onnx/onnx/issues/2904
2. Fix and enable Windows conda packaging and install, which has been broken since 2018, https://github.com/onnx/onnx/issues/2612

**Comment by glenn-jocher** [comment]
Hardswish support would be a welcome addition in 1.8 https://github.com/onnx/onnx/issues/2728

**Comment by sveta-levitan** [comment]
ONNX Training work is tracked in https://github.com/onnx/onnx/issues/2886. There are several PR's in progress and some tasks that have not started.

**Comment by daquexian** [comment]
1. ONNX Optimizer work is tracked in https://github.com/onnx/optimizer/issues/1 

2. Softmax family ops: https://github.com/onnx/onnx/pull/2879

**Comment by jcwchen** [comment]
Hi ONNX contributors,
Milestone for ONNX 1.8 has been added in the ONNX repo. If you have work items which need to go in 1.8.0 release, please add milestone 1.8 to your PR before 10th Oct. Thank you.

**Comment by padmasreenagarajan** [comment]
> Hardswish support would be a welcome addition in 1.8 #2728

Yes..
In Mobilenetv3+SSD,
**h-swish** uses 4 separate operations like **add,clip,multiply and div** given by the following lines of code

**out =  F.hardtanh(x + 3, 0., 6.) / 6**
**return  out*x**
      
Since it is repeated multiple times in the network model, it would be better to add this operator in ONNX==1.8 release

**Comment by glenn-jocher** [comment]
I'll add this here also just for clarity. While it's technically possible to create a manual hardswish op today in ONNX, it requires 4 seperate ops, repeated about 100 hundred times in YOLOv5, once per activation layer, resulting in a greatly expanded list of operations, which makes visualization cumbersome as well.

> Current hardswish op in onnx using 4 separate operations, repeated about a hundred times in a YOLOv5 model.
> 
> ```python
>         return x * F.hardtanh(x + 3, 0., 6.) / 6.  # for torchscript, CoreML and ONNX
> ```
> 
> <img alt="Screen Shot 2020-09-23 at 10 56 00 AM" width="876" src="https://user-images.githubusercontent.com/26833433/94050779-93ca1f80-fd8b-11ea-9a9b-52c2a6701809.png">

**Comment by padmasreenagarajan** [comment]
> I'll add this here also just for clarity. While it's technically possible to create a manual hardswish op today in ONNX, it requires 4 seperate ops, repeated about 100 hundred times in YOLOv5, once per activation layer, resulting in a greatly expanded list of operations, which makes visualization cumbersome as well (and may have performance implications?)
> 
> > Current hardswish op in onnx using 4 separate operations, repeated about a hundred times in a YOLOv5 model.
> > ```python
> >         return x * F.hardtanh(x + 3, 0., 6.) / 6.  # for torchscript, CoreML and ONNX
> > ```
> > 
> > 
> > <img alt="Screen Shot 2020-09-23 at 10 56 00 AM" width="876" src="https://user-images.githubusercontent.com/26833433/94050779-93ca1f80-fd8b-11ea-9a9b-52c2a6701809.png">

Has the PR  been created for h-swish in ONNX==1.8.0 release? @glenn-jocher

**Comment by glenn-jocher** [comment]
@padmasreenagarajan I don't know. I maintain YOLOv5, I'm unrelated to the ONNX team.

**Comment by padmasreenagarajan** [comment]
> I'll add this here also just for clarity. While it's technically possible to create a manual hardswish op today in ONNX, it requires 4 seperate ops, repeated about 100 hundred times in YOLOv5, once per activation layer, resulting in a greatly expanded list of operations, which makes visualization cumbersome as well (and may have performance implications?)
> 
> > Current hardswish op in onnx using 4 separate operations, repeated about a hundred times in a YOLOv5 model.
> > ```python
> >         return x * F.hardtanh(x + 3, 0., 6.) / 6.  # for torchscript, CoreML and ONNX
> > ```
> > 
> > 
> > <img alt="Screen Shot 2020-09-23 at 10 56 00 AM" width="876" src="https://user-images.githubusercontent.com/26833433/94050779-93ca1f80-fd8b-11ea-9a9b-52c2a6701809.png">

Has the PR already been created for h-swish in ONNX==1.8.0?
@glenn-jocher

**Comment by jcwchen** [comment]
Hi ONNX contributors,
Since we are still working on opset 13 validation before ONNX Release, we postpone the date for 1.8 PyPI package testing (for Community to test) until next Monday (10/26) for now. Thank you.

**Comment by padmasreenagarajan** [comment]
@jcwchen 
Can we raise a PR now for ONNX=1.8.0 release

**Comment by jcwchen** [comment]
Hi @padmasreenagarajan, 
Sorry that we will cut the release branch for ONNX 1.8.0 soon so the answer is no. Thank you

**Comment by jcwchen** [comment]
1.9 is coming. Close this one now.
